File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/order/exception/OrderItemOptionChangedException.java
package band.gosrock.domain.domains.order.exception;


import band.gosrock.common.exception.DuDoongCodeException;

public class OrderItemOptionChangedException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new OrderItemOptionChangedException();

    private OrderItemOptionChangedException() {
        super(OrderErrorCode.ORDER_OPTION_CHANGED);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/order/exception/OrderItemOptionChangedException.kt
package band.gosrock.domain.domains.order.exception

import band.gosrock.common.exception.DuDoongCodeException

class OrderItemOptionChangedException private constructor() : DuDoongCodeException(OrderErrorCode.ORDER_OPTION_CHANGED) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = OrderItemOptionChangedException()
    }
}
